dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,185 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[introduction]]
+
+== Introduction
+
+Using the Jenkins OpenShift Client Plugin, you can quickly and easily create Jenkins Pipelines utilizing the OpenShift DSL
```

Comment:
DSL -> fluent DSL

i.e. let's highlight / tout the "sugary fluent-ness" of the syntax

Reply:
I don't see a mention of the phrase "fluent DSL" anywhere in the existing OpenShift docs repo, I only see one place where it mentions fluent, and that's in the Jenkins Client Plugin repo

"DSL Plugin is a Jenkins plugin which aims to provide a readable, concise, comprehensive, and fluent ".  

If it makes sense I can add some language like that to this section, but I hesitate to update everywhere it says DSL to say "fluent DSL"

Reply:
updated in next push

Reply:
technically we didn't do much of anything in the opnshift docs for the client plugin since it was experimental, so in a general sense I would say its this PR that is beefing up the doc to GA status

As I noted later below, yeah, changing DSL to fluent DSL everywhere is probably overboard, but we should highlight in the client plugin intro that is is fluent styled, etc.

Cool deal re: update in next push ... that said, if he has bandwidth, we should solicit @jupierce 's opinion as client plugin creator on whether we should highlight to some degree the "fluent" nature of the plugin ... as I recall, syntax style was one of the original motivators.